week-1/K-functions-parameters/exercise5.js

```diff
@@ -1,4 +1,6 @@
-// Declare your function here
+function createLongGreeting (name, age) {
+    return "Hello my name is " + name + " and I'm " + age + " years old";
+}
 
 const greeting = createLongGreeting("Daniel", 30);
```

Comment:
Good trying new things like const as opposed to var, be sure to understand the difference if you don't yet know 👍